content/en/docs/more/maintenance.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Maintenance
+weight: 80
+aliases:
+  - /maintenance
+---
+
+This document explains how maintenance and updates are performed on Darcy Cloud.
+
+There are two broad categories of maintenance:
+
+- **Darcy Cloud maintenance**: Darcy Cloud [Portal](https://cloud.darcy.ai)
+  and [API](https://api.darcy.ai).
+- **Node maintenance**: updates to the agent software and/or
+  configuration installed on your edge devices.
+
+Typical Darcy Cloud maintenance is invisible to the end user. That is to say,
+there is no interruption to service, and you may not have noticed that the update
+has occurred. We refer to this as a [Zero-Impact Maintenance](#zero-impact-maintenance) event.
+However, on occasion
+we may need to temporarily interrupt service to perform significant infrastructure
+or backend upgrades. We call this a [Service-Impact Maintenance](#service-impact-maintenance) event.
+
+During a maintenance window, you can learn about service availability
+at [status.darcy.ai](https://status.darcy.ai).
+
+## Impact
+
+### Zero-Impact Maintenance
+
+Zero-Impact Maintenance, as you might expect, has zero impact on service availability.
+Most often, this type of maintenance is to improve backend services or infrastructure.
+If there are user-facing changes, you will be able to find information
+on those changes in the [changelog](/docs/more/release-notes).
+
+### Service-Impact Maintenance
+
+During _Service-Impact Maintenance_, there is interruption to service.
+
+- Darcy Cloud Portal will be unavailable. A visit to [cloud.darcy.ai](https://cloud.darcy.ai) in
+  your browser will return an "Undergoing Maintenance" page.
+- Darcy Cloud API routes will return a `503/Service Unavailable` HTTP status code.
```

Comment:
There is a discussion in Slack about the return message with Nora about not showing 503 if it is not really a 503 error (however Mozilla says a 503 can represent maintenance).

Reply:
I saw that. We'll be returning 503 from CloudAPI, which is the most appropriate status code.